drivers/hmis/app/models/hmis/concerns/hmis_arel_helper.rb

```diff
@@ -25,6 +25,10 @@ def fd_t
       Hmis::Form::Definition.arel_table
     end
 
+    def fi_t
```

Comment:
do you want this down on line 70 as well?

Reply:
IDK I'm not sure why that's really there. I was just copying ArelHelper which duplicates everything in class methods and outside. It doesn't seem like its needed